### PR TITLE
WIP: Fedora CI

### PR DESCRIPTION
**Pull request by bkmgit**
Initial version to do tests on Fedora using podman

### CODE REVIEW

**Comment by bkmgit** [comment]
Can squash commits, but initial feedback would be helpful if you wish to add this.

May want to clone using checkout action and then copy the repository into the container.

File: .github/workflows/fedora.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Test on Fedora
+on:
+  - push
+
```
Comment (heronshoes) [inline .github/workflows/fedora.yml:4]: It didn't tested on pull request. How about to add `pull_request:` at `on:` ?

File: .github/workflows/fedora.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        run: |
+          echo 'FROM fedora:38' >> podmanfile
+          echo 'RUN dnf -y update' >> podmanfile
+          echo 'WORKDIR /' >> podmanfile
+          echo 'RUN dnf -y install git gcc-c++ libarrow-devel libarrow-glib-devel ruby-devel' >> podmanfile
+          echo 'RUN dnf clean all' >> podmanfile
+          echo 'RUN git clone https://github.com/heronshoes/red_amber.git' >> podmanfile
+          echo 'WORKDIR /red_amber' >> podmanfile
+          echo 'RUN bundle install' >> podmanfile
+          echo 'RUN TZ=Asia/Tokyo bundle exec rake test' >> podmanfile # set TZ to ensure the test using timestamp
```
Comment (heronshoes) [inline .github/workflows/fedora.yml:26]: It may be better to split `TZ=Asia/Tokyo` to ENV statement.
(If it doesn't seem simple, you can leave it as is.)

**Comment by bkmgit** [comment]
Closed in favor of https://github.com/heronshoes/red_amber/pull/151

File: .github/workflows/fedora.yml (hunk 2 above)
Comment (bkmgit) [reply .github/workflows/fedora.yml]: Done.

File: .github/workflows/fedora.yml (hunk 1 above)
Comment (bkmgit) [reply .github/workflows/fedora.yml]: Done.

**Pull request closed without merging**